l10n.md

```diff
@@ -0,0 +1,48 @@
+# Translating Ladybird
+## gettext
+Ladybird is translated with the help of [gettext](https://www.gnu.org/software/gettext/).
+
+Add a new translation:
+
+### 1. Add a new po file
+
+You can find current translations in https://github.com/bugaevc/ladybird-gtk4/tree/master/po, copy one of the files and translate all the strings `msgstr` **[what is the command the user should run instead of copying one of the current files?]**
+
+You also have to change some meta information:
+```
+# Swedish translation for Ladybird.
+"Language: sv\n"
+"Plural-Forms: nplurals=2; plural=n != 1;\n"
+```
+Read more about [Plural-Forms](https://www.gnu.org/software/gettext/manual/html_node/Plural-forms.html)
+
+### 2. CMakeLists.txt
+
+Add your .po-file to [/po/CMakeLists.txt](https://github.com/bugaevc/ladybird-gtk4/blob/master/po/CMakeLists.txt)
+```
+    set(linguas
+        eo.po
+        ru.po
+        sv.po
+    )
+```
+### 3. org.serenityos.Ladybird-gtk4.desktop
+Translate the strings in [org.serenityos.Ladybird-gtk4.desktop](https://github.com/bugaevc/ladybird-gtk4/blob/master/org.serenityos.Ladybird-gtk4.desktop)
+
+```
+GenericName[sv]=Webbläsare
+Name[sv]=Nytt fönster
+Name[sv]=Nytt privat fönster
+```
+
+## Translating Adwaita components
+
+Some components in Ladybird, like `Tab Overview` ([TabOverview](https://gnome.pages.gitlab.gnome.org/libadwaita/doc/main/class.TabOverview.html)) and `About Ladybird` ([AboutWindow](https://gnome.pages.gitlab.gnome.org/libadwaita/doc/main/class.AboutWindow.html)) uses components from Adwaita. These might or might not already have a translation, you can find these in [https://gitlab.gnome.org/GNOME/libadwaita/-/tree/main/po](https://gitlab.gnome.org/GNOME/libadwaita/-/tree/main/po).
+
+
+## Resources for translating
+
+GNOME Wiki might have information about localization & translating your language, give `https://wiki.gnome.org/{Language name in English}` a look. (e.g. https://wiki.gnome.org/Esperanto, https://wiki.gnome.org/Swedish).
+
+If libadwaita is already translated to your language then feel free to reuse terms and words for consistency with Adwaita/GNOME https://gitlab.gnome.org/GNOME/libadwaita/-/tree/main/po
```

Comment:
When in doubt, it's also a good idea to look at how the similar component / string is translated in Epiphany. We don't follow Epiphany 100%, but it's also a very handy resource for design & translation ideas.